platform/ui/src/components/Button/Button.tsx

```diff
@@ -46,7 +46,7 @@ const variants = {
       'text-secondary-light hover:opacity-80 active:opacity-100 focus:opacity-80',
     white: 'text-white hover:opacity-80 active:opacity-100 focus:opacity-80',
     black:
-      'text-black hover:bg-primary-main focus:bg-primary-main hover:border-black focus:border-black',
+      'text-white hover:bg-primary-main focus:bg-primary-main hover:border-black focus:border-black',
```

Comment:
This should be text-black - the specific behaviour is that this is specified by the color attribute of the button, which specifies the text color for use - specifically look at line 47, which has text-white.  If it is desire to use white, then specify white as the annotation.

Reply:
This is actually a problem with the PR we recently changed, there are various places in UI that are broken as a result, my recommendation is to revert that change (which was an oversight from me since I didn't check all the components), and a new PR that probably add a new variant for the app you are working on. How about that?